.vscode/launch.json

```
@@ -1,6 +1,15 @@
 {
     "version": "0.2.0",
     "configurations": [
+        {
+            "name": "Python test.py",
+            "cwd": "${workspaceFolder}",
```

I think the test needs to be executed in the "BadgeFramework" folder for the imports to work properly
```suggestion
            "cwd": "${workspaceFolder}/BadgeFramework",
```